src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Operations/Import/ImportRequestExtensionsTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Api.Features.Operations.Import;
+using Microsoft.Health.Fhir.Core.Features.Operations.Import.Models;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Shared.Api.UnitTests.Features.Operations.Import
+{
+    public class ImportRequestExtensionsTests
+    {
+        [Fact]
+        public void GivenImportReuqestInParamtersFormat_WhenConvert_ThenImportRequestShouldBeReturned()
```

Comment:
Nit: typo 'Reuqest'. It is also in the other test name.

Reply:
thanks, updated.